### PR TITLE
Up to question 30 for 1426 PDF

### PR DESCRIPTION
**Pull request by jessieay**
Part of [#152727221]

Signed-off-by: Jessie Young <jessie@cylinder.work>

### CODE REVIEW

File: db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class RenameEmployedMonthlyEmployerToEmployerEmployerNames < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :employed_employer_names, :string, array: true
```
Comment (jessieay) [inline db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb]: renamed these fields to make their meaning clearer.

Just had a call with @rachelcope where I confirmed that in the next iteration of the "amounts - income" page, we are going to ask for employment pay quantities and intervals, so nothing related to monthly income

File: db/schema.rb [hunk 2]
```diff
@@ -161,11 +161,11 @@
     t.boolean "disabled"
     t.boolean "employed", default: false
     t.string "employed_employer_name"
+    t.string "employed_employer_names", default: [], array: true
```
Comment (bengolder) [inline db/schema.rb]: Do you think these should eventually be pulled out into a separate table?

File: db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb (hunk 1 above)
Comment (bengolder) [reply db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb]: This feels like an improvement, but I still find it a little confusing to read `employed_employer`. would it be clearer if it were `job_employer_names` or `employment_employer_names`? 
It seems like a noun in place of the `employed` adjective would make it clear that this is a property of a thing, which I think is implied.

File: db/schema.rb (hunk 2 above)
Comment (jessieay) [reply db/schema.rb]: yes perhaps! now that we have employer data that is different for SNAP and medicaid this is all getting pretty confusing.

File: db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb (hunk 1 above)
Comment (jessieay) [reply db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb]: yeahhhh -- the reason I named it this way was that we prepend ever `job` field with `employed` and every self employment field with `self_employed` -- I am not sure this pattern is even that helpful at this point, but it is an existing pattern that provides some means of remembering which fields are named what. Do you think we should move away from that pattern, or perhaps do that when moving this data to a separate table?

**Pull request merged**